sample/Kmd.Logic.Gateway.Automation.Sample/Program.cs

```diff
@@ -65,8 +65,6 @@ private static async Task Run(AppConfiguration configuration)
             {
                 Console.WriteLine(result.ToString());
             }
-
-            Console.WriteLine("WIP");
         }
 #pragma warning restore CA1031
```

Comment:
remove this warning disable, I think it is no more required

Reply:
Removed